python/pdstools/adm/ADMDatamart.py

```diff
@@ -152,6 +153,8 @@ def __init__(
         verbose: bool = False,
         **reading_opts,
     ):
+        self.logger = logging.getLogger(__name__)
```

Comment:
I don't think you should set a logger to an object? Isn't the convention to set a logger at file level (i.e., just set `logger=logging.getLogger(__name__)` and simply do `logger.info()`?